Resources/Prototypes/Reagents/medicine.yml

```diff
@@ -43,7 +43,7 @@
       - !type:HealthChange
         damage:
           types:
-            Poison: -1
+            Poison: -1.5 # Was 1, Slight Buff as it should heal for half the amount as Dip or Stelli(GoobStation)
```

Comment:
Perhaps look into buffing diphenhydramine and stelli by 1.5-2x? Considering that replacing a liver (incredibly easy) now heals it instantly, you have to think about either nerfing that or changing to match.

Reply:
The current state of livers magically healing all poison damage is not intended to be the norm, just a funny and easy replacement for dialysis/blood pumps while i figure out how to deal with the procedures that would depend on a static structure (IV drips, dialysis, respirators, pumps, etc). 

But a buff would probably be in order anyway since they will be as efficient as having tend wounds for those specific damage types.

Reply:
Diphenhydramine Heals 90u poison per 15u, Setllibinnin Heals 96 poison per 15u. I don't think a buff should be necessary for those at the time being.